Sources/PenguinTesting/StdlibProtocolTests.swift

```diff
@@ -188,13 +201,11 @@ fileprivate class SequenceChecker<Subject, ExampleContents>
   /// The sequence under test
   var subject: Subject
 
-  /// Elements either expected in, or to be written into, `subject`
+  /// Elements either expected in `subject`
```

Comment:
This change doesn't make sense to me; the thing no longer parses as English (and we both left out the trailing period).

Reply:
Resolved by completely reverting the `checkDeclaredSequenceRefinementSemantics`